templates/todo/api/csharp-sql-func/ListsFunctions.cs

```diff
@@ -7,13 +7,13 @@
 namespace SimpleTodo.Api;
 public class ListsFunctions
 {
-    private readonly ILogger _logger;
-    private readonly ListsRepository _repository;
+    private readonly ILogger logger;
```

Comment:
Can you verify that all c# files in our repo across all templates follow this convention?